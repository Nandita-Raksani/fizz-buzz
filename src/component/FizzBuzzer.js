import React from 'react';

const FizzBuzzer = (props) => {

    const doFizzBuzz = () => {
      const number = props.number;
      return 1;
    }
    
  return (
      <div>      
        <form>
            <label>{doFizzBuzz()}</label>                
        </form>
      </div>
  );
}
export default FizzBuzzer;