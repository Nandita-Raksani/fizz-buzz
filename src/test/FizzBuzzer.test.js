import React from 'react';
import FizzBuzzer from "../component/FizzBuzzer";
import { shallow, configure } from 'enzyme';

describe(("<FizzBuzzer/> component"), () => {
    let wrapper;
    beforeEach(() => {
      wrapper = shallow(<FizzBuzzer/>);
    });
  
    it("should render correctly", () => {
      expect(wrapper).toMatchSnapshot();
    });
    
  });

  describe(("print functionality"), () => { 
    it("should print 1 for prop number passed as 1", () => {
        let wrapper = shallow(<FizzBuzzer number="1"/>);
        expect(wrapper.find('label').text()).toBe("1");
    });
  });