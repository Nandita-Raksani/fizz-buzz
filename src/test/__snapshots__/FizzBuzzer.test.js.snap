// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FizzBuzzer/> component should render correctly 1`] = `ShallowWrapper {}`;
